js/main.js

```diff
@@ -0,0 +1,391 @@
+			icon: '<i class="fas fa-wheat"></i>',
+			recipe: '&health=wheat-free',
+			menu: '&intolerances=wheat'
+		},
+*/
+		'Peanut-Free': {
+			name: 'Peanut-Free',
+			icon: '<i class="fas fa-acorn"></i>',
+			recipe: '&health=peanut-free',
+			menu: '&intolerances=peanut'
+		},
+		'Tree-Nut-Free': {
+			name: 'Tree-Nut-Free',
+			icon: '<i class="fas fa-tree-alt"></i>',
+			recipe: '&health=tree-nut-free',
+			menu: '&intolerances=tree-nut'
+		},
+		'Vegan': {
+			name: 'Vegan',
+			icon: '<i class="fas fa-carrot"></i>',
+			recipe: '&health=vegan',
+			menu: '&diet=vegan'
+		},
+		'Vegetarian': {
+			name: 'Vegetarian',
+			icon: '<i class="fas fa-salad"></i>',
+			recipe: '&health=vegetarian',
+			menu: '&diet=vegetarian'
+		}
+    };
+    
+    // create content section   
+	var content = $('<section>').attr('id', 'content');
+	var contentContainer = $('<div>').addClass('container');
+	
+	// create map container
+	var map = $('<section>').attr('id', 'map');	
+    
+    // create footer   
+	var footer = $('<footer>').attr('id', 'container');
+	footer.html('<div class="row"><div class="col s12"><h2>Didn’t find what you’re looking for?</h2><a href="#hero" class="btn-large deep-orange lighten-2">Search Again</a></div></div>');
+	
+/*	
+	// pull food joke API
+	var jokeURL = "https://api.spoonacular.com/food/trivia/random?apiKey=e0a3536a362b46d38d50a5b045964f5a";
+
+	$.ajax({
+	  url: jokeURL,
+	  method: "GET"
+	}).then(function(response) {
+		console.log('Food Joke API Response: ', response.text);
+	});
+*/
+
+	
+	// on form submit...
+	$('#hero form').on('submit', function(event) {
+		event.preventDefault();	 
+		
+		// clear html each time to prevent multiple populating
+		$('#content .container').html('');
+		
+		// only run if search has a value, otherwise show alert
+		if ($('#food-type').val() === '') {
+			$('div#hero-content .alert').html('Search field cannot be blank.');
+			return;
+		}
+		
+		// hide alert once it runs
+		$('div#hero-content .alert').html('&nbsp;');
+		
+		// pull food type value
+		var foodType = $('#food-type').val();
+		
+		// create empty strings to fill in for loop
+		var iconString = ''
+		var recipeQueryString = '';
+		var menuQueryString = '';		
+			
+		// pull choices and create an array from it
+		var restrictionChoices = $('#restrictions input').val();
+		var restrictionArray = restrictionChoices.split(', ');
+		
+		// if restrictions aren't blank, update strings
+		if (restrictionChoices !== 'Dietary Restrictions') {
+			// for each form option selected, add elements to strings
+			restrictionArray.forEach(function(i) {			
+				iconString = iconString.concat(restrictions[i].icon);
+				recipeQueryString = recipeQueryString.concat(restrictions[i].recipe);
+				menuQueryString = menuQueryString.concat(restrictions[i].menu);			
+			});
+		}
+
+		// pull recipe API
+	    var recipeURL = 'https://api.edamam.com/search?app_id=d544ae9f&app_key=c5ad09c117643ee56f64724e79d6a318&to=12' + '&q=' + foodType;	    
```

Comment:
this is just a code style thing, these lines are a little long

when you start getting to the point where you might have to scroll, you want to start using multiple lines

pluses should also start the line, so 

```'https://api.edamam.com/search?app_id=d544ae9f&app_key=c5ad09c117643ee56f64724e79d6a318&to=12' 
+ '&q='
+ foodType;```